src/components/CalendarEventsDialog.vue

```diff
@@ -117,6 +156,45 @@ async function getCalendars() {
 					<p>{{ loading ? t('spreed', 'Loading …') : t('spreed', 'No upcoming events') }}</p>
 				</template>
 			</NcEmptyContent>
+
+			<div v-if="canScheduleMeeting" class="calendar-meeting">
+				<h4 class="calendar-meeting__header">
+					{{ t('spreed', 'Schedule a meeting') }}
+				</h4>
+				<div class="calendar-meeting__flex-wrapper">
+					<NcSelect id="schedule_meeting_select"
+						v-model="selectedCalendar"
+						:options="calendarOptions"
+						:placeholder="t('spreed', 'Select calendar …')"
+						:input-label="t('spreed', 'Select calendar')">
+						<template #option="option">
+							<span class="calendar-badge" :style="{ backgroundColor: option.color }" />
+							{{ option.label }}
+						</template>
+					</NcSelect>
+					<NcDateTimePickerNative id="schedule_meeting_input"
+						v-model="selectedDateTime"
+						:label="t('spreed', 'Meeting start time')"
+						type="datetime-local" />
+				</div>
+				<NcTextField v-model="newMeetingTitle"
+					:label="t('spreed', 'Name')"
+					:placeholder="t('spreed', 'Name')"
```

Comment:
In the calendar app it's called "Event title" maybe we go with title then instead?

Reply:
aligned with Calendar app